modules/oadp-configuring-node-agent-load-concurrency.adoc

```diff
@@ -0,0 +1,63 @@
+
+* `globalConfig`: Defines a default concurrency limit for the Node Agent across all nodes.
+* `perNodeConfig`: Specifies different concurrency limits for specific nodes based on `nodeSelector` labels. This provides flexibility for environments where certain nodes might have different resource capacities or roles.
+
+
+.Prerequisites
+To control the rate at which requests are sent to the Velero server, which directly impacts its ability to handle concurrent operations, you need to configure specific fields within your DPA custom resource (CR):
+
+* **OADP Operator Installation**: The OpenShift API for Data Protection (OADP) Operator must be installed on your {product-title} to configure these settings. You must be logged in as a user with `cluster-admin` privileges to install the OADP Operator.
+* **DPA CR Configuration**: The Velero Server Kubernetes API queries per second (QPS) and burst values are managed through the `spec.configuration.velero.client-qps` and `spec.configuration.velero.client-burst` fields within your DPA.
+** The `client-burst` setting determines how many requests can be sent to the Velero server before a limit is applied.
+** After the burst limit is reached, the `client-qps` setting determines how many additional requests can be sent per second.
+** These values default to 100 in OADP 1.4.1 and later. Velero's default client burst and QPS values changed from 30/20 to 100/100 in Velero 1.14, which is the version installed by OADP 1.4. You can adjust these values, for example, setting `client-burst: 500` and `client-qps: 300`.
+
+
+.Procedure
+
+. If you want to use Load Concurrency for specific nodes, add labels to those nodes:
++
+[source,terminal]
+----
+$ oc label node/<node_name> label.io/location='EU'
+----
+
+. Configure the Load Concurrency fields for your DPA instance:
++
+[source,yaml]
+----
+  configuration:
+    nodeAgent:
+      enable: true
+      loadConcurrency:
+        globalConfig: 1 # <1>
+        perNodeConfig:
+        - nodeSelector:
+            matchExpressions:
+            - key: label.io/location # <2>
+              operator: In
+              values:
+              - EU # <3>
+          number: 3 # <4>
+----
+<1> Specify the `globalConfig` value. The default value is 1, which means there is no concurrency and only one load is allowed. The `globalConfig` value does not have a limit.
+<2> Specify the `key` value that you used to add labels.
+<3> Specify the values you used to add labels.
+<4> Specify the `number` value for per-node concurrency. You can specify a different concurrent number per node. For example, you can set 3 concurrent instances for `EU` and 2 instances for `US`. The range of per-node concurrent number is the same as the global concurrent number. If the configuration file contains a per-node concurrent number and a global concurrent number, the per-node concurrent number takes precedence.
```

Comment:
I think better example on the load concurrency and labeling would be around the instance type and not the location to show the users that the load concurrency should be applied for instance type and size. so:

```yaml
  configuration:
    nodeAgent:
      enable: true
      loadConcurrency:
        globalConfig: 1
        perNodeConfig:
          - nodeSelector:
              matchLabels:
                 label.io/instance-type: large
            number: 3

```